### PR TITLE
chore: clean branch with latest AP-888

### PR DESCRIPTION
**Pull request by joshua-barnard-evolv**

### CODE REVIEW

**Comment by joshua-barnard-evolv** [comment]
I opted to include checks for the new context items. I think it's a fair test since we want to be certain those were the items in the new context.

File: helpers/ExperimentDetails_helper.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    async getCurrentActiveKeys() {
+        const {Puppeteer} = this.helpers;
+
+        return await Puppeteer.executeScript(async () => {
+            const keys = await evolv.client.getActiveKeys();
+            return keys.current;
+        })
+    }
+
+    updateContext(contextItems) {
```
Comment (robertsevernsentient) [inline helpers/ExperimentDetails_helper.js:29]: Not blocking, but updateContext and getContextItems don't really do anything anymore -- they call directly through to the evolv client and could be removed

File: helpers/ExperimentDetails_helper.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    /**
+     * @protected
+     */
+    _after() {
+        // remove if not used
+    }
+
+    async getCurrentActiveKeys() {
```
Comment (robertsevernsentient) [inline helpers/ExperimentDetails_helper.js:20]: Is there a file missing? I'm not seeing you use these helpers in the existing tests

**Review by robertsevernsentient — APPROVED** [review]

File: helpers/ExperimentDetails_helper.js (hunk 2 above)
Comment (joshua-barnard-evolv) [reply helpers/ExperimentDetails_helper.js:20]: As per our chat, this is the first use of the helpers. I'll use them to refactor the other tests.

File: helpers/ExperimentDetails_helper.js (hunk 1 above)
Comment (joshua-barnard-evolv) [reply helpers/ExperimentDetails_helper.js:29]: As per our chat, Puppeteer is required to access `evolv` on the page.

**Pull request merged**